src/Components/NavLink/NavLink.tsx

```diff
@@ -2,13 +2,17 @@ import React from 'react';
 
 import { default as NavLinkStyles} from '../Nav/Nav.module.scss';
 import bemCssModule from 'bem-css-modules';
-
+import { Link } from '../../Constants/types';
 const style=bemCssModule(NavLinkStyles);
 
-const NavLink = ({title, to}:any) => {
+export interface LinkItem {
+    link:Link
+}
+
+const NavLink: React.FC<LinkItem> = ({link}:LinkItem) => {

```

Comment:
Try to keep correct names  if you typed component.  LinkItem -> NavLinkProps


= ({link}) =>